### ISSUE TITLE
I cannot simulate the halo effcet by the psf you provided

### ISSUE THREAD

**Issue by fishing-pan** [jnjaby/DISCNet#23]
In the paper, we  can see noticeable hole around light, but I cannot simulate the halo effcet by the psf files(npy format) and your data_simulation.py code you provided.

**Comment by jnjaby** [comment]
Hi, did you simulate with HDR images? As stated in our paper, a common problem causing incomplete degradation is the simulation with LDR images.

**Comment by fishing-pan** [comment]
您好，您的邮件我已收到，我将尽快回复您，祝您工作顺利，身体健康，开开心心！

**Comment by fishing-pan** [comment]
> Hi, did you simulate with HDR images? As stated in our paper, a common problem causing incomplete degradation is the simulation with LDR images.

How to distinguish LDR image and HDR image ,I load a 8bits sRGB image by OpenCV. 
Additionally，I opened the training dataset your provided(npy files), but there are too dark, did I need do some pre-processing? such as Normalized.

**Comment by jnjaby** [comment]
To visualize HDR images, you need to convert them into SDR (Standard Dynamic Range) via tone mapping. Please check out [tone mapping function](https://github.com/jnjaby/DISCNet/blob/63b1859519091f8790afcc47e8c726cbefdcd0fe/basicsr/data/paired_img-psf_npy_dataset.py#L68).

8-bit sRGB images are not suitable for our simulation. I suggest you referring to Fig. 3 in the supplementary file, which could be the case.

**Comment by fishing-pan** [comment]
I get it, thanks for your reply.
